python/ray/tune/analysis/experiment_analysis.py

```diff
@@ -88,10 +87,10 @@ class NewExperimentAnalysis:
     def __init__(
         self,
         experiment_checkpoint_path: Union[str, os.PathLike],
+        storage_filesystem: Optional[pyarrow.fs.FileSystem] = None,
```

Comment:
Since we're reordering already
```suggestion
        *,
        storage_filesystem: Optional[pyarrow.fs.FileSystem] = None,
```